.github/workflows/test.yml

```
@@ -51,6 +60,8 @@ jobs:
           cpu_architecture: ${{matrix.arch}}
           source_directory: ${{matrix.package}}
           artifacts_directory: artifacts
+          lintian_opts: ${{matrix.lintian_opts}}
+          lintian_run: ${{matrix.lintian_run}}
```

```suggestion
          lintian_run: ${{matrix.lintian_run || 'false' }}
```

Maybe this will work